### ISSUE TITLE
Remove or shade/hide guava dependency

### ISSUE THREAD

**Issue by spkrka** [spotify/folsom#121]
This would possibly reduce potential dependency problems.

**Comment by ayoukhananov** [comment]
@spkrka Any progress ?

**Comment by spkrka** [comment]
No, sorry. Looks like you have implemented this shading in your fork - you could perhaps submit a PR here with the same shading configuration?

**Comment by ayoukhananov** [comment]
Ok, I will submit a PR on it .

**Comment by ayoukhananov** [comment]
@spkrka Please review : https://github.com/spotify/folsom/pull/161
